### PR TITLE
Tweak Coursier installation instructions

### PR DESCRIPTION
**Pull request by dwijnand**
A miscellaneous bag of changes. Feedback welcome.

### CODE REVIEW

File: readme/Installation.scalatex [hunk 1]
```diff
@@ -10,30 +10,31 @@
...
 
-        @p
-          @b{Recommended in CI/Continuous Integration Server}. Put the bootstrap
-          script in your code repository to make sure everyone on your team use
-          the same scalafmt version. To configure which files to format,
-          see @sect.ref{project}.
+      @p
+        Create a standalone executable in @code{/usr/local/bin/scalafmt} with (sudo if necessary):
+        @hl.xml
+          coursier bootstrap com.geirsson:scalafmt-cli_2.12:@V.stable \
```
Comment (olafurpg) [inline readme/Installation.scalatex]: I'm surprised --main is no longer needed, at some point it complained about an ambiguous

File: readme/Installation.scalatex [hunk 2]
```diff
@@ -10,30 +10,31 @@
...
+            -o /usr/local/bin/scalafmt --standalone
+          scalafmt --version # should be @org.scalafmt.Versions.stable
 
+      @p
+        Alternatively you can create a slim 15 KiB bootstrap script with:
```
Comment (olafurpg) [inline readme/Installation.scalatex]: 👍 much clearer

File: readme/Installation.scalatex [hunk 3]
```diff
@@ -10,30 +10,31 @@
...
+        @hl.xml
+          coursier bootstrap com.geirsson:scalafmt-cli_2.12:@V.stable \
+            -o scalafmt
+          ./scalafmt --version # should be @org.scalafmt.Versions.stable
+
+        It is @b{recommended} to put this bootstrap script in your code repository
```
Comment (olafurpg) [inline readme/Installation.scalatex]: good to clarify this!

File: readme/Installation.scalatex (hunk 1 above)
Comment (dwijnand) [inline readme/Installation.scalatex]: Interesting to know. I wondered what the default behaviour was without the flag and found the behaviour to work as intended. So I proposed dropping it.

**Pull request merged**